test/helpers/openParticipantsPane.js

```diff
@@ -0,0 +1,11 @@
+const ParticipantsPane = require("../page-objects/ParticipantsPane");
+const Toolbox = require("../page-objects/Toolbox")
+export default async function openParticipantsPane() {
+    const toolbox = await Toolbox.ToolboxView;
```

Comment:
This reads super weird because you are awaiting a property. Please call it something like `getXXX`